bin/cli.js

```diff
@@ -9,34 +9,54 @@ const Version = {
     MAJOR: 'major'
 };
 
-function getParentBranch() {
-    return execSync('hg log --rev "p2(.)" --template "{branch}"');
+function getParentBranches() {
+    const currentRev = execSync('hg id -i').toString().trim();
+    const currentBranch = execSync('hg id --branch').toString().trim();
+    const lastestTag = execSync(`hg log -r "." -b ${currentBranch} --template "{latesttag}"`).toString().trim();
+
+    let branches = [];
+    if (lastestTag) {
```

Comment:
I think it would be worth it that you add a plain english comment for line 19 and 21 explaining what it does